modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/DatabaseReaderLazyLoader.java

```diff
@@ -194,6 +198,15 @@ int current() {
     }
 
     DatabaseReader get() throws IOException {
+        if (lastUpdate != 0) {
```

Comment:
maybe add a comment that only managed dbs have a `lastUpdate` that is unequal to zero,
and that therefore for config and builtin dbs this checks never happens?

Reply:
Added